### ISSUE TITLE
[bug] isDevTools evals code unsafely

### ISSUE THREAD

**Issue by aulneau** [tamagui/tamagui#361]
**Describe the bug**

There is some code in `/core` that evals javascript, specifically `isDevTools`:
```js
const isDevTools = process.env.NODE_ENV === "development" ? new Function("try {return this===window;}catch(e){ return false;}")() : false;
```

When trying to use tamagui in the context of a web extension (manifest v3), eval is not allowed in any context. 

Here is the patch i used to get it working (includes some changes for getConfig like in https://github.com/tamagui/tamagui/commit/a701eb3f8c83f96d9ddce419388fdd710b8fa6c4:

````diff
diff --git a/dist/esm/constants/isDevTools.js b/dist/esm/constants/isDevTools.js
index b7ea1e0739b7a6a8405e7cfd97d1c2bf5c8b5d01..aadae34efce958fe1e2596ebbd5f33e693b05871 100644
--- a/dist/esm/constants/isDevTools.js
+++ b/dist/esm/constants/isDevTools.js
@@ -1,4 +1,4 @@
-const isDevTools = process.env.NODE_ENV === "development" ? new Function("try {return this===window;}catch(e){ return false;}")() : false;
+const isDevTools = false;
 export {
   isDevTools
 };
diff --git a/dist/esm/constants/isDevTools.js.map b/dist/esm/constants/isDevTools.js.map
index dd0c7522397afe14c9ac7e5a34993c9379be9456..b9127f928af63d8a6b5f5070c6dad921aceafd11 100644
--- a/dist/esm/constants/isDevTools.js.map
+++ b/dist/esm/constants/isDevTools.js.map
@@ -1,7 +1,7 @@
 {
   "version": 3,
   "sources": ["../../../src/constants/isDevTools.ts"],
-  "sourcesContent": ["// for verbose logging outside node/cli\n\nexport const isDevTools =\n  process.env.NODE_ENV === 'development'\n    ? new Function('try {return this===window;}catch(e){ return false;}')()\n    : false\n"],
+  "sourcesContent": ["// for verbose logging outside node/cli\n\nexport const isDevTools = false\n"],
   "mappings": "AAEO,MAAM,aACX,QAAQ,IAAI,aAAa,gBACrB,IAAI,SAAS,qDAAqD,EAAE,IACpE;",
   "names": []
 }
diff --git a/dist/esm/views/TamaguiProvider.js b/dist/esm/views/TamaguiProvider.js
index b513d629bf9fa57034dcb9f064e4bfe1f6037c3b..4d65b4372d9d98e04d352d31e77ab8ddea1101f7 100644
--- a/dist/esm/views/TamaguiProvider.js
+++ b/dist/esm/views/TamaguiProvider.js
@@ -5,7 +5,6 @@ import { ButtonInsideButtonContext } from "../contexts/ButtonInsideButtonContext
 import { TextAncestorContext } from "../contexts/TextAncestorContext";
 import { useMediaListeners } from "../hooks/useMedia";
 import { ThemeProvider } from "./ThemeProvider";
-import { getConfig } from "..";
 function TamaguiProvider({
   children,
   disableInjectCSS,
@@ -35,7 +34,6 @@ function TamaguiProvider({
       };
     }, [config, disableInjectCSS]);
   }
-  const { themes } = getConfig();
   return /* @__PURE__ */ jsx(ButtonInsideButtonContext.Provider, {
     value: false,
     children: /* @__PURE__ */ jsx(TextAncestorContext.Provider, {
@@ -44,7 +42,7 @@ function TamaguiProvider({
         themeClassNameOnRoot: config.themeClassNameOnRoot,
         disableRootThemeClass: config.disableRootThemeClass,
         ...themePropsProvider,
-        defaultTheme: themePropsProvider.defaultTheme ?? Object.keys(themes)[0],
+        defaultTheme: themePropsProvider.defaultTheme ?? Object.keys(config.themes)[0],
         children
       })
     })
````

**Comment by natew** [comment]
Fixed in 6d1ad48c38bf3cd14babfc0d5865dc2c9a044a0e (release in a minute)

**Comment by aulneau** [comment]
dope -- just tested it out, works!! thanks for the quick fix :)
